helpers/response/create_response.py

```diff
@@ -1,52 +1,51 @@
 from datetime import datetime
-from graphql import GraphQLError
+# from graphql import GraphQLError
```

Comment:
Remove the commented out code from this file

Reply:
Done